ui/components/ui/actionable-message/index.scss

```diff
@@ -14,7 +14,7 @@
   }
 
   &--with-icon.actionable-message--warning {
-    justify-content: normal;
+    justify-content: center;
```

Comment:
What's the impact of this on the other components where this is used? 
Is it certain that this change will not affect the other pages in a way that is not suitable?

Reply:
@david0xd I researched the impact of this on the other components in app and you're right about this. 
I changed it immediately, sorry, my mistake.
You can see changes here: 68e87f995e68cad0d644b5e3a68139d0432df9d4
Is it okay now?